crates/core/src/repofile/snapshotfile.rs

```
@@ -1307,6 +1308,16 @@ impl PathList {
         self.0.clone()
     }
 
+    /// Clone the internal `Vec<PathBuf>`.
+    #[must_use]
+    pub(crate) fn unix_paths(&self) -> Vec<UnixPathBuf> {
+        self.0
+            .iter()
+            .map(|p| p.clone().try_into())
+            .collect::<Result<_, _>>()
+            .unwrap()
```

Instead of `unwrap`, couldn't we warn and ignore path that fail to become `unix_path`? 
Or error out with a `RusticError`?